bindings/matrix-sdk-ffi/CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 <!-- next-header -->
 
 ## [Unreleased] - ReleaseDate
+- Adjust features in the `matrix-sdk-ffi` crate to expose more platform-specific knobs.
```

Comment:
```suggestion

- Adjust features in the `matrix-sdk-ffi` crate to expose more platform-specific knobs.
```

Reply:
Please add a category to this as well, refactor or feature.

Reply:
Sorry, took me a while to understand (since the matrix-sdk-ffi Changelog doesn't have these sub-headers).  I added a new section header of `### Refactor`